shopinvader_cart_reminder/data/mail_template.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would put this template in the demo data. It is only used in your test and I do not see the added value for the final user.

Reply:
Ok but then we should suggest it as default value on the backend otherwise users are not aware of its existence

Reply:
@Cedric-Pigeon you're right... ping @acsonefho